packages/config/src/package.schema.json

```
@@ -0,0 +1,62 @@
+{
+  "type": "object",
+  "definitions": {
+    "caretOnlyVersion": {
+      "type": "string",
+      "pattern": "^\\^"
+    },
+    "workspaceOnlyVersion": {
+      "type": "string",
+      "pattern": "(^\\*$)|(^file:)"
+    },
+    "caretOrWorkspaceOnlyVersion": {
+      "type": "string",
+      "pattern": "(^\\^)|(^\\*$)|(^file:)"
+    },
```

We use `npm` and probably stay with it, so this is not needed right now, but could be worth to add here sooner or later. At least I leave a comment for future reference 🙂 

With `pnpm` and `yarn` we would need to add `^workspace:`

```suggestion
    "workspaceOnlyVersion": {
      "type": "string",
      "pattern": "(^\\*$)|(^file:)|(^workspace:)"
    },
    "caretOrWorkspaceOnlyVersion": {
      "type": "string",
      "pattern": "(^\\^)|(^\\*$)|(^file:)|(^workspace:)"
    },
```